src/SCRIPTS/BF/pages.lua

```diff
@@ -56,4 +56,8 @@ if apiVersion >= 1.41 then
     PageFiles[#PageFiles + 1] = { title = "GPS PIDs", script = "gpspids.lua" }
 end
 
+if apiVersion >= 1.16 then
+    PageFiles[#PageFiles + 1] = { title = "Trim Accelerometer", script = "acc_calibrate.lua" }
```

Comment:
maybe "acc_trim.lua" would be a more appropriate filename since we already have similarly named files to calibrate the accelerometer.